crates/stackable-operator/src/cli.rs

```diff
@@ -381,16 +418,17 @@ mod tests {
 
     #[test]
     fn product_operator_run_watch_namespace() {
-        // clean env var to not interfere if already set
-        unsafe { env::remove_var(WATCH_NAMESPACE) };
-
         // cli with namespace
         let opts = ProductOperatorRun::parse_from([
             "run",
             "--product-config",
             "bar",
             "--watch-namespace",
             "foo",
+            "--operator-namespace",
+            "stackable-operators",
+            "--operator-service-name",
+            "foo-operator",
```

Comment:
note: I still feel like all these CLI unit tests are pretty much useless and should be removed to speed up the testing runs.